config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -3,14 +3,13 @@ en:
     assessment:
       submission:
         submissions:
+          finalise: 'Finalise Submission'
+          publish: 'Publish Grade'
+          auto_grade: 'Auto Grade Submission'
           create:
             failure: 'Could not create submission: %{error}'
           edit:
             attempt: :'course.assessment.assessments.assessment.attempt'
-          worksheet:
```

Comment:
I would have kept this, then `finalse: :'course.assessment.submission.finalise'` etc etc.